packages/wallet-sdk/src/CoinbaseWalletSDK.ts

```diff
@@ -87,33 +65,27 @@ export class CoinbaseWalletSDK {
     return walletLogo(type, width);
   }
 
-  private get walletExtension(): LegacyProviderInterface | undefined {
-    return window.coinbaseWalletExtension;
-  }
+  private grabInjectedProvider(): ProviderInterface | undefined {
```

Comment:
how about having this as a separate util function?
doesn't seem like it uses any internal methods of this class anyway.
and, what would be a better name for this?

Reply:
udpated